modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -504,6 +508,18 @@ private Model getParent(Model model) {
 
     }
 
+    private void removeReadOnly(Map<String, Object> modelTemplate) {
+        if (modelTemplate != null
+                && modelTemplate.containsKey("model")
+                && modelTemplate.get("model") instanceof CodegenModel) {
+            CodegenModel model = (CodegenModel) modelTemplate.get("model");
+
+            if (model != null && model.vars != null) {
+                model.vars.forEach((var) -> var.isReadOnly = false);
```

Comment:
That would work too though it's then a change that has to be applied per language and potentially library.  It would at least solve my problem with updating petstore 😄